test/pylib/start_mock.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
```

Comment:
the name of `start_mock.py` is way too general. what it does is to start a mock s3 server. so i'd suggest rename this script to `start_s3_mock.py`.

Reply:
Done